RELEASE_NOTES.md

```diff
@@ -1,3 +1,21 @@
+### 0.11.3 January 12 2022 ####
+
+* Add deserialization whitelisting feature [#281](https://github.com/akkadotnet/Hyperion/pull/281)
```

Comment:
I'd reword this to "allow explicit control over which types can be deserialized" - and then I'd mention that it's recommended to use this feature as a best practice going forward because it can prevent https://cwe.mitre.org/data/definitions/502.html

I'll probably also need to add security notices for this repo too so that alert will get pushed out in the dependabot notifications for repos that have it enabled. I'll need to look into how to do that.